src/Test/Utilities/Portable/TestUtilities.csproj

```diff
@@ -95,6 +95,7 @@
     <PackageReference Include="xunit" Version="$(xunitVersion)" />
     <PackageReference Include="xunit.analyzers" Version="$(xunitanalyzersVersion)" />
     <PackageReference Include="xunit.runner.console" Version="$(xunitrunnerconsoleVersion)" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="$(xunitrunnervisualstudioVersion)" />
```

Comment:
❓ Why is this added? This project does not contain tests.

Reply:
Oh, I was expecting the reference to be in one of our shared **.targets** files, not here...

Reply:
Let's add this to `build\Targets\Roslyn.Toolsets.Xunit.targets`. This targets file is imported to all unit test projects.

Reply:
While you're changing this please also move `xunit` and `xunit.runner.console` there.
Also, `xunit.analyzers` do not need to be explicitly reference since they are referenced by `xunit`.

Reply:
> Also, xunit.analyzers do not need to be explicitly referenced since they are referenced by xunit.

We reference a newer version of the analyzers than would be brought in automatically, so the reference is still needed.

Reply:
Ok, then let's move that reference to `build\Targets\Roslyn.Toolsets.Xunit.targets` as well

Reply:
I disagree. We've made an explicit decision in the past to put all our package references in project files, not target files. This is for a couple reasons, but mainly it's to prevent references from being magically included or excluded based on target configurations. I think we should put these in the project files, like we do with all of our other package references.

Reply:
@jaredpar I believe you had strong feelings on this. Care to weigh in?

Reply:
I tried this and it doesn't work. The "TestUtilities" projects don't import the Roslyn.Toolsets.Xunit.targets file because the targets file is only for actual unit test projects. Technically, the runner isn't necessary in the utilities projects either, so just the runner references could be added to the targets file, but this means that half the xunit references are in project files and half the references are in targets files. This is more confusing than either choice. I think we should stick with the original design.

Reply:
The runner should not be referenced by non-test projects. TestUtilities are not a test project. They should only reference the xunit packages they need (xunit.abstractions i think).

Reply:
The console runner is already referenced. This works fine.